pyproject.toml

```diff
@@ -29,6 +29,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.9,<4"
+pathlib = "^1.0.1"
```

Comment:
```suggestion
```
pathlib is builtin, so we don't want an external dependency.

Reply:
removed path lib from dependencies